content/en/agent/kubernetes/apm.md

```diff
@@ -134,6 +156,16 @@ List of all environment variables available for tracing within the Agent running
 | `DD_APM_MAX_EPS`           | Sets the maximum Analyzed Spans per second. Default is 200 events per second.                                                                                                                                                                                                                                               |
 | `DD_APM_MAX_TPS`           | Sets the maximum traces per second. Default is 10 traces per second.                                                                                                                                                                                                                                                        |
 
+### Operator environment variables
+| Environment variable       | Description                                                                                                                                                                                                                                                                                                                 |
+| -------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `agent.apm.enabled`                                                                                          | Enable this to enable APM and tracing, on port 8126 ref: https://github.com/DataDog/docker-dd-agent#tracing-from-the-host                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
you should make these actual page level links with the "For a reference, see..." because how this is doesn't actually make it a clickable url in the docs - someone would have to copy paste it